src/components/checkbox/checkbox.scss

```diff
@@ -42,6 +42,7 @@
     }
 
     label {
+        cursor: pointer;
```

Comment:
When I first saw the PR title, I thought you had made the label look like a link… 😂

I've updated the commit message to:

> fix(checkbox): use cursor style `pointer` to indicate label can be clicked

Reply:
😸